### ISSUE TITLE
Feature request: Easy to find OWL data for a single GO term

### ISSUE THREAD

**Issue by JervenBolleman** [geneontology/helpdesk#485]
For example I would like to see the OWL modeling of the term [GO:0006040](https://amigo.geneontology.org/amigo/term/GO:0006040) easily from the term page. 

While I would like OWL in any of the RDF syntaxes plus manchester, others would want OBO and some JSON thingy.

**Comment by balhoff** [comment]
@JervenBolleman you can find this in the OLS view: https://www.ebi.ac.uk/ols4/ontologies/go/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FGO_0006040

It would be nice for us to have a link there from Amigo. What do you think of that @kltm?

**Comment by JervenBolleman** [comment]
I can only find a JSON option on OLS not an OWL or OBO option.

**Comment by balhoff** [comment]
Oh I see, you would like a machine-readable download per term? I was thinking you just wanted to see the OWL modeling, which on that page is in Manchester syntax (in HTML).

**Comment by JervenBolleman** [comment]
@balhoff yes (although in my case it is for human consumption). The OWL model is so incomplete that I had not considered it. Basically I wanted to see how term Y regulates term X was modeled so that I can use it a query.

**Comment by balhoff** [comment]
@JervenBolleman can you say more about what you would want to see for a particular term that would be more complete? OLS has its equivalent class expression as well as its superclasses. Are you interested in a bigger picture of the OWL modeling in GO? Just trying to understand what would be useful. We have some OWL design patterns described here: https://github.com/geneontology/go-ontology/blob/master/docs/patterns/index.md  For example this is the regulation pattern: https://github.com/geneontology/go-ontology/blob/master/docs/patterns/regulation.md
